.circleci/config.yml

```diff
@@ -132,6 +132,18 @@ jobs:
             echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
             npm --no-git-tag-version version `node build/get-version.js`-`git rev-parse --short HEAD`
             npm publish --tag=staging
+  deploy:
+    working_directory: ~/cp-zen-frontend
+    docker:
+      - image: alpine
+    steps:
+      - trigger_downstream_job: ./update_platform.sh latest master
```

Comment:
where is `trigger_downstream_job` in circle docs?

Reply:
Derp, I'm an idiot..........

Reply:
```
run:
     - name: trigger_downstream_job
     - command: ./update_platform.sh latest master
```
i assume is what you want but you also need to 
` - checkout ` first